plugins/inputs/ps.go

```diff
@@ -1,4 +1,4 @@
-package system
+package inputs
```

Comment:
Can you leave this file in the system package for the time being?  Long term I would like to split it out to the required plugins as well as splitting the interface up, but we should do that after moving the files to reduce the size of the conflict window.